tests/fixtures/requirements-locked.txt

```
@@ -32,3 +32,7 @@ tomli @ https://files.pythonhosted.org/packages/97/75/10a9ebee3fd790d20926a90a25
 -e git+ssh://git@github.com/phylum-dev/phylum-ci.git@7d6d859ad368d1ab0a933f24679e3d3c08a40eac#egg=phylum
 
 -e /tmp/editable ; python_version >= "3.7" and python_version < "3.12"
+
+--index-url https://mirror1.phylum.io/simple/
```

```suggestion
-i https://mirror1.phylum.io/simple/
```
Not actually suggesting this line be changed...but just showing how if it was, the test would fail.